src/Statement/Filter/StatementFilter.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Wikibase\DataModel\Services\Statement\Filter;
+
+use Wikibase\DataModel\Statement\Statement;
+
+/**
+ * @since 3.2
+ *
+ * @licence GNU GPL v2+
+ * @author Thiemo Mättig
```

Comment:
It filters statements. There is only 1 public method. It requires 1 parameter and returns a bool. I do not know how more obvious it can be. What would a description add to this? What do you suggest?